src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = {
+                new Ball(), new Ball(), new Ball()
+        };
+        for (Ball object : balls) {
+            lottery.getRandomBall(object);
+            System.out.println(object);
```

Comment:
should be like that. add required changes to the Lottery#getRandomBall method so it will create new ball, fill it with random field values and then will return it
```suggestion
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
```